src/msg/async/AsyncMessenger.cc

```diff
@@ -171,13 +171,16 @@ void Processor::accept()
     Worker *w = worker;
     if (!msgr->get_stack()->support_local_listen_table())
       w = msgr->get_stack()->get_worker();
+    else
```

Comment:
can we put 
```c++
if (msgr->get_stack()->support_local_listen_table()) {
  w = worker;
  w->reference;
} else {
  w = msgr->get_stack()->get_worker();
}
```

1. use positive logic to help with the readablity
1. more explicit on what the `w` is assigned
1. post-increment is faster for `atmic<>` types